lib/ledger_sync/adaptors/quickbooks_online/deposit/operations/create.rb

```diff
@@ -0,0 +1,61 @@
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Deposit
+        module Operations
+          class Create < Operation::Create
+            class Contract < LedgerSync::Adaptors::Contract
+              params do
+                required(:ledger_id).value(:nil)
+                required(:to_account).hash(Types::Reference)
+                required(:currency).filled(:string)
+                required(:memo).filled(:string)
+                required(:transaction_date).filled(:date?)
+                required(:exchange_rate).maybe(:float)
+                required(:line_items).array(Types::Reference)
+              end
+            end
+
+            private
+
+            def operate
+              response = adaptor.post(
+                resource: 'deposit',
+                payload: local_resource_data
+              )
+
+              resource.ledger_id = response.dig('Id')
+              success(response: response)
+            end
+
+            def local_resource_data
+              {
+                'CurrencyRef' => {
+                  'value' => resource.currency,
+                },
+                'TxnDate' => resource.transaction_date.to_s, # Format: YYYY-MM-DD
+                'PrivateNote' => resource.memo,
+                'ExchangeRate' => resource.exchange_rate,
+                'DepositToAccountRef' => {
+                  'value' => resource.to_account.ledger_id
+                },
+                'Line' => resource.line_items.map do |line_item|
+                  {
+                    'DetailType' => 'DepositLineDetail',
+                    'DepositLineDetail' => {
+                      'AccountRef' => {
+                        'value' => line_item.account&.ledger_id || resource.to_account.ledger_id
+                      }
+                    },
+                    'Amount' => line_item.amount / 100.0,
+                    # 'Description' => line_item.description
```

Comment:
@jozefvaclavik why is this commented out

Reply:
Ups, sigh.